### ISSUE TITLE
Got nan Regression loss and inf Classification loss on pytorch 1.5

### ISSUE THREAD

**Issue by manaclan** [qfgaohao/pytorch-ssd#121]
Hi, I reused your code on pytorch 1.5 but after few hundreds examples the loss suddenly explode. I have read many issue in this repo and this [issue](https://github.com/qfgaohao/pytorch-ssd/issues/21) seem relevant.  
Is there a way to fix the problem? Thanks

**Comment by ijalalfrz** [comment]
hi @manaclan have you resolved your issue, because i get same result

**Comment by manaclan** [comment]
hi, I've move to this repo and trained successfully, you can give it a try https://github.com/amdegroot/ssd.pytorch
